app/src/main/java/com/keylesspalace/tusky/util/CustomEmojiHelper.kt

```diff
@@ -45,7 +46,9 @@ fun CharSequence.emojify(emojis: List<Emoji>?, view: View) : CharSequence {
     if(emojis.isNullOrEmpty())
         return this
 
-    val builder = SpannableStringBuilder.valueOf(this)
+    val builder = SpannableString.valueOf(this)
+    val pm = PreferenceManager.getDefaultSharedPreferences(view.context)
+    val animate = pm.getBoolean(PrefKeys.ANIMATE_CUSTOM_EMOJIS, false)
```

Comment:
The setting should be queried once per Activity and passed into the function as a Boolean param, as we try to do with other settings. `StatusDisplayOptions` is used in most places for this.

Reply:
Correct. Gonna change it then.